src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml

```diff
@@ -23,13 +23,6 @@
   tags:
     - py3
 
-- name: Add Python PPA for bionic and xenial
-  apt_repository: repo='ppa:jonathonf/python-3.6' state=present
```

Comment:
I think we need to proactively remove this from all machines too. If we just remove this what version of python will get installed on new machines? 

The pango one (ppa:jonathonf/gtk3.18) also needs to be replaced.

Reply:
```
# python3 --version
Python 3.6.9
# python --version
Python 2.7.17
```

Reply:
OK, that's fine.